### ISSUE TITLE
A11y issue: Image detail screen is missing a title

### ISSUE THREAD

**Issue by diederv** [parley-messaging/android-library#80]
Image detail screen is missing a title

![Image](https://github.com/user-attachments/assets/7283c3f2-7c2f-43ad-9fe5-d8d5f238a8c3)

**Comment by esmitswebuildapps** [comment]
![Image](https://github.com/user-attachments/assets/c3b479e4-67b1-48c7-85f0-5d7a01a86217)
Added a filename to the image viewer
